engine/settings/base.py

```diff
@@ -509,6 +509,7 @@ class BrokerTypes:
 )
 
 CELERY_BEAT_SCHEDULE_FILENAME = os.getenv("CELERY_BEAT_SCHEDULE_FILENAME", "celerybeat-schedule")
+CELERY_BEAT_SYNC_ORGANIZATIONS_INTERVAL_MINUTES = os.getenv("CELERY_BEAT_SYNC_ORGANIZATIONS_INTERVAL_MINUTES", 5)
```

Comment:
Wondering if we should start with a more conservative interval?

Reply:
I thought we can give it a shot at this frequency. Since they are spread out the worst case still is an org is at the beginning in one task an then the end of another resulting in 9 mins between sync. Maybe that can be adjusted see next comment.